DXMainClient/DXGUI/Multiplayer/GameInformationPanel.cs

```diff
@@ -84,11 +119,21 @@ public override void Initialize()
                 lblPlayerNames[(lblPlayerNames.Length / 2) + i] = lblPlayerName2;
             }
 
+            pingTextures = new Texture2D[5]
+            {
+                AssetLoader.LoadTexture("ping0.png"),
```

Comment:
Will the client work normally when these assets are missing?

Reply:
These are the same ping images used in the cncnetgamelobby